keps/sig-network/20190603-EndpointSlice-API.md

```diff
@@ -179,6 +180,13 @@ type Endpoint struct {
     //   endpoint is located. This should match the corresponding node label.
     // +optional
     Topology map[string]string `json:"topology,omitempty" protobuf:"bytes,5,opt,name=topology"`
+    // TerminationGracePeriodSeconds is the termination grace period (in seconds) the
+    // endpoint is given to gracefully shutdown. Consumers should use this field to determine
+    // how long existing connections to an endpoint should be kept open after the endpoint
+    // is deleted.
+    // +optional
+    TerminationGracePeriodSeconds *int64 `json:"terminationGracePeriodSeconds,omitempty" protobuf:"bytes,6,opt,name=terminationGracePeriodSeconds"`
```

Comment:
@robscott mentioned that another alternative would be to add a `Terminating` field in `EndpointCondition` (https://github.com/kubernetes/kubernetes/pull/88590#discussion_r387391601). But that would require some larger changes in the endpointslice controller